services/telemetry/telemetry.go

```diff
@@ -1110,3 +1163,94 @@ func (s *Server) trackChannelModeration() {
 		"use_channel_mentions_guest_disabled_count": useChannelMentionsGuest,
 	})
 }
+
+func (ds *TelemetryService) initSegmentIO(endpoint string) {
```

Comment:
why is there still segment related code? I believe it's already replaced bu rudder 🤔

Reply:
I don't know, that is a question for @grundleborg

Reply:
Putting a do not merge label on this PR to block it until https://github.com/mattermost/mattermost-server/pull/14712 is through QA and merged, as it is considerably higher priority than this (sorry Jesus).

Reply:
Ohh, sure, first things first :)